examples/phi2/README.md

```diff
@@ -1,13 +1,38 @@
 # Phi-2
 
+## Install the onnxruntime-genai library
+
+* (Temporary) Build the library according to [the build instructions](../README.md#build-from-source)
+
+* Install the python package
+
+  ```bash
+  cd build/wheel
+  pip install onnxruntime-genai-*.whl
+  ```
+
 ## Get the model
 
+Install the model builder script dependencies
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
+Run the model builder script to load, optimize, quantize and export the model. More details can be found [here](../../src/python/py/models/README.md)
+
+```bash
+cd examples/llama
+python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o ./model
```

Comment:
```suggestion
python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o ./phi2-int4-cpu
```

Reply:
If users try to run multiple examples, then the shared `model` folder may contain files from multiple examples and that may cause unforeseen issues during runtime. Naming the output folder with a specific name in each example would put the files in separate folders and help prevent this.

Reply:
That's fair. I'll update it

Reply:
Updated